config/packages/prod/monolog.yaml

```diff
@@ -4,14 +4,14 @@ monolog:
             type: fingers_crossed
             action_level: error
             handler: nested
-            excluded_404s:
-                # regex: exclude all 404 errors from the logs
-                - ^/
+            excluded_http_codes: [404, 405]
+            buffer_size: 50 # How many messages should be saved? Prevent memory leaks
         nested:
             type: stream
-            path: "php://stderr"
+            path: php://stderr
             level: warning
+            formatter: monolog.formatter.json
```

Comment:
@fabiocicerchia it's ok for you?

Reply:
as long as the logs are printed to stdout it is fine by me :)